content/influxdb/cloud-dedicated/admin/custom-partitions/define-custom-partitions.md

```diff
@@ -41,37 +41,48 @@ Use the following command flags to identify
 
 - `--template-tag`: An [InfluxDB tag](/influxdb/cloud-dedicated/reference/glossary/#tag)
   to use in the partition template.
-  _Supports up to seven of these flags._
+- `--template-tag-bucket`: An [InfluxDB tag](/influxdb/cloud-dedicated/reference/glossary/#tag)
+  and number of "buckets" to group tag values into.
+  Provide they tag key and the number of buckets to bucket tag values into
+  separated by a comma: `tagKey,N`.
 - `--template-timeformat`: A [Rust strftime date and time](/influxdb/cloud-dedicated/admin/custom-partitions/partition-templates/#time-part-templates)
   string that specifies the time format in the partition template and determines
   the time interval to partition by.
 
+{{% note %}}
+A partition template can include up to 7 total tag and tag bucket parts
+and only 1 time part.
```

Comment:
Oh TIL... should I add this to the CLI help output as well?

Reply:
I wouldn't hurt. If they try to do more than 8 template parts, the server returns an error.